inst/qml/msaGaugeRR.qml

```diff
@@ -92,7 +93,7 @@ Form
 			label:								qsTr("Type 3 study (automatic equipment)")
```

Comment:
`Type` -> `type3` (camelCasing)

Reply:
but this is a label, not an option name? or are you referring to something else?

Reply:
Ah, nevermind, I see now what you mean

Reply:
Yeah sorry, I could have been clearer. I misplaced this suggest because GitHub does not let me comment in PRs on lines that have _not_ been changed